shards/shards_test.go

```diff
@@ -226,22 +226,25 @@ func TestShardedSearcher_Ranking(t *testing.T) {
 	}
 }
 
-func TestFilteringShardsByRepoSet(t *testing.T) {
+func TestFilteringShardsByRepoSetOrBranchesReposRepoIds(t *testing.T) {
```

Comment:
This test was already testing both `RepoSet` and `BranchesReops` so I hijacked it and added a 3rd.

Reply:
LGTM. Starting to look like a win32 API :P